functions/v2/cloud-event-logging/index.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_structured_logging_event]
+const cloudLogging = require('@google-cloud/logging');
+const functions = require('@google-cloud/functions-framework');
+
+functions.cloudEvent('structured_logging_event', async cloudEvent => {
+  const logging = new cloudLogging.Logging();
+  const log = logging.logSync('Structured_logging_event project');
+  const text = 'Hello, world!';
+
+  const metadata = {
+    severity: 'NOTICE',
+    component: 'arbitrary-property',
+    cloudEvent: cloudEvent,
+  };
+
+  const entry = log.entry(metadata, text);
+  log.write(entry);
```

Comment:
The goal of this sample is to show "syntactic sugar" and provide a separate sample for code that doesn't have a request object. cloudEvent is not parsed by the library like httpRequest is for trace propagate. Though this brings up an interesting idea of adding all the cloud event info to the log may be useful.
```
const entry = log.entry({"arbitrary": "field"}, "Hello, world!");
log.error(entry)
```